src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,176 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The Ui class deals with interactions with the user.
+ */
+public class Ui {
+    private static final String INDENTATION = "    ";
```

Comment:
Perhaps you can use a common prefix for all 3 constants as they serve similar functions? (eg. printing out to console)